template/cordova/lib/AppxManifest.js

```diff
@@ -374,6 +374,11 @@ AppxManifest.prototype.getCapabilities = function () {
 };
 
 function refineColor(color) {
+    if (color.indexOf('0x') === -1 && color.indexOf('#') === -1) {
```

Comment:
WIndows SDK will do the validation and omit errors for invalid values and it's not for us to validate here.